aiidalab_widgets_base/computational_resources.py

```diff
@@ -858,6 +866,17 @@ def _configure_computer(self, computer: orm.Computer):
 
         return True
 
+    def _configure_computer_local(self, computer: orm.Computer):
+        """Configure the computer with local transport"""
+        authparams = {
+            "use_login_shell": self.use_login_shell.value,
+            "safe_interval": self.safe_interval.value,
+        }
+        user = orm.User.collection.get_default()
```

Comment:
Maybe do that inside `_configure_computer` method and then pass the `user` argument to `_configure_computer_local` and `_configure_computer_ssh`? We will then use it also for the other transport types.